Send All.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE MudletPackage>
+<MudletPackage version="1.001">
+    <TriggerPackage/>
+    <TimerPackage/>
+    <AliasPackage>
+        <Alias isActive="yes" isFolder="no">
+            <name>Global Command</name>
+            <script>send(matches[2])
+raiseGlobalEvent(&quot;my_event&quot;, matches[2])
+</script>
+            <command></command>
+            <packageName></packageName>
+            <regex>^:(.*)$</regex>
+        </Alias>
+    </AliasPackage>
+    <ActionPackage/>
+    <ScriptPackage>
+        <Script isActive="yes" isFolder="no">
+            <name>Global Command</name>
+            <packageName></packageName>
+            <script>function my_function(_, text)
```

Comment:
IIRC the `raiseGlobalEvent` call appends an extra argument which is the profile name of the _sending_ profile - does that get correctly handled (or discarded) by the `registerAnonymousEventHandler` call?

Reply:
Boo, what @SlySven said is exactly what happened. If I type ":follow ansrare", "Ansrare follow ansrare" is echoed in all consoles.

Reply:
OT: @Ansrare - you have a testing profile devoted to yourself? 😏 

Though - assuming the sending profile is called `Ansrare` it seems surprising that it is being shown before the first - but that might just be the behaviour of the handler...

Reply:
I'm fairly unoriginal when it comes to names, so when I find something I like I use it all.the.places.

Yes, the sending profile is Ansrare. I also thought it was strange that it led with that. It also *looks* like it's echoing, not sending, but I'm not sure how to tell for sure (my mud says "huh?" if it doesn't recognize a command, and it didn't reply at all after "Ansrare follow ansrare").